web/src/pages/_app.tsx

```diff
@@ -28,8 +28,8 @@ import { StyledEngineProvider, ThemeProvider } from "@mui/material";
 import "@newjersey/njwds/dist/css/styles.css";
 import { DefaultSeo } from "next-seo";
 import { AppProps } from "next/app";
-import Head from "next/head";
 import { useRouter } from "next/compat/router";
+import Head from "next/head";
```

Comment:
automated import sorting from a [prettier-plugin](https://github.com/simonhaenisch/prettier-plugin-organize-imports)

Reply:
[dust] Our VSCode settings also provide this functionality.

Reply:
I noticed this issue when working in another file, and it appears to be some conflict between our [vscode setting](https://github.com/newjersey/navigator.business.nj.gov/blob/main/.vscode/settings.json#L22) and the prettier plugin we're using, even though [we're actually disabling that plugin for this file](https://github.com/newjersey/navigator.business.nj.gov/blob/246ca6975ea92409b2845a77f0c4998a25802cc4/web/src/pages/_app.tsx#L2) 🤷